Bookmarks/providers/edge.py

```diff
@@ -0,0 +1,42 @@
+# Keypirinha: a fast launcher for Windows (keypirinha.com)
+
+import pyesedb
+import keypirinha_util as kpu
+import os.path
+from ._base import Bookmark, BookmarksProviderBase
+
+class EdgeProvider(BookmarksProviderBase):
+    localappdata_dir = None
+
+    SPARTAN_FILE_LOCATION = r"Packages\Microsoft.MicrosoftEdge_8wekyb3d8bbwe\AC\MicrosoftEdge\User\Default\DataStore" \
+                            r"\Data\nouser1\120712-0049\DBStore\spartan.edb"

```

Comment:
I have no clue but this path looks like it could be different among multiple environments?

Reply:
Surprisingly enough, the path is probably fixed. The same one is mentioned [here](https://answers.microsoft.com/en-us/edge/forum/edge_other-edge_win10/location-of-microsoft-edge-favorites/ff89cc47-0597-44ad-b2f5-3505f8d5ae85)